test/streaming-client.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { PassThrough } from 'stream';
+import { WebSocketClient, WebSocketConnection } from 'websocket';
+import { AudioConfig } from '../src/models/streaming/AudioConfig';
+import { BufferedDuplex } from '../src/models/streaming/BufferedDuplex';
+import { RevAiStreamingClient } from '../src/streaming-client';
+const fs = require('fs');
+const events = require('events');
+
+jest.useFakeTimers();
```

Comment:
are we using this?

Reply:
Yes, we call runOnlyPendingTimers on line 150